webhook/resourcesemantics/defaulting/controller.go

```diff
@@ -74,7 +86,7 @@ func NewAdmissionController(
 	}
 
 	logger := logging.FromContext(ctx)
-	c := controller.NewImpl(wh, logger, "DefaultingWebhook")
```

Comment:
why can't the work queue name be `"DefaultingWebhook-"+name` ?

Reply:
That would work and make webhook match the behaviour of our other controllers

Reply:
It would work. FWIW, this is not only used for the queue name but also the lease name for lead election. By changing this, a lease object will be leaked after upgrading an existing cluster. Are we fine with this?

This is for https://github.com/knative/pkg/issues/1979. @n3wscott @dprotaso are you fine with the solution to modify upstream code? Matt prefers solution 3 mentioned in https://github.com/knative/pkg/issues/1979#issuecomment-755814906.

Reply:
See my comments here:

https://github.com/knative/pkg/issues/1979#issuecomment-797855624